ADDONS.md

```diff
@@ -129,6 +129,7 @@ module RubyLsp
     end
 
     # All listeners have to inherit from ::RubyLsp::Listener
+    # NOTE: Listeners are currently being refactored, but this implementation embodies the current approach for most listeners.
```

Comment:
Thought it would be helpful to add a note here since the code below will no longer be accurate for `Hover`

Reply:
Yeah, we'll need to update all of the docs. It might be worth waiting until the refactors are complete and then we update them once.